packages/datetime/src/common/dateUtils.ts

```diff
@@ -110,6 +110,27 @@ export function isMonthInRange(date: Date, dateRange: DateRange) {
     return start <= day && day <= end;
 }
 
+export function isTimeInRange(time: Date, minTime: Date, maxTime: Date): boolean {
```

Comment:
call the first arg `date` since it doesn't need to be time-only (and frees up the name `time` for comment below).